package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "stackoverflow-api",
+  "version": "1.0.0",
+  "description": "```sh\r  A user should create an account and login\r ```\r ```sh\r A user should be able to create a post.\r ```\r ```sh\r A user should be able to change his or her post.\r ```\r ```sh\r A user should be able to delete a post.\r ```\r ```sh\r A user should be able to view the created post\r ```\r ```sh\r A user should be able to view all the created posts.\r ```\r ```sh\r A user should be able to search for a post\r ```",
```

Comment:
The description should describe what the app does and not the user stories as you presented here.
for example:  An app to manage blog posts